README.md

```diff
@@ -160,4 +160,4 @@ If you use OceanBioME as part of your research, teaching, or other activities, w
 }
 ```
 
-To cite a specific version of the package please also cite its [Zenode archive](https://doi.org/10.5281/zenodo.10038575).
+To cite a specific version of the package please also cite its [Zenodo archive](https://doi.org/10.5281/zenodo.10038575).
```

Comment:
If we choose to keep it then replace with the generic doi link (as I suggested above)? Otherwise we need to update the zenodo link with every release which is a logistical nightmare?

Reply:
Hi @navidcy @jagoosw, let me know if I need to commit this suggestion.

Reply:
let's leave @jagoosw decide on this

Reply:
I thought we should include something to let people know they can cite a specific version for reproducibility of their work if they wanted so maybe it should be reworded to "you may cite..." and let them know they can get the specific DOI?